tracer.go

```diff
@@ -169,7 +169,7 @@ func (t *Tracer) TraceQueryEnd(ctx context.Context, _ *pgx.Conn, data pgx.TraceQ
 	span := trace.SpanFromContext(ctx)
 	recordError(span, data.Err)
 
-	if data.Err != nil {
+	if data.Err == nil {
```

Comment:
Good spot! Could you also change this in line 207 below?

Reply:
Of course, sorry for the late reply, it's done!